code-style/dependency-check-suppressions.xml

```diff
@@ -267,4 +267,24 @@
         <packageUrl regex="true">^pkg:maven/net\.minidev/json-smart@.*$</packageUrl>
         <vulnerabilityName>CVE-2024-57699</vulnerabilityName>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+        These libraries are dependencies of spark-sql. Awaiting versions of libraries without the vulnerability.
+        file name: RoaringBitmap-1.3.0.jar
+        file name: shims-0.9.45.jar
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.roaringbitmap/(RoaringBitmap|shims)@.*$</packageUrl>
+        <cve>CVE-2018-17073</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+        The following JSON libraries have a stack overflow issue that allows for DDoS attacks.
+        Once there's a version of any of these libraries that don't, we can use those instead.
+        file name: aws-json-protocol-2.31.59.jar
+        file name: json-utils-2.31.59.jar
+        file name: json-unit-core-2.36.0.jar
```

Comment:
It's not clear from the notes why we're using a vulnerable version of json-unit-core.

The AWS ones are because the latest version of the AWS SDK still has the vulnerabilities.

The json-unit-core one is because WireMock uses it, and we haven't upgraded WireMock because we're on an old version of trino-testing, which uses the old version of WireMock.